Jot/TrackingConfiguration.cs

```diff
@@ -40,6 +40,12 @@ public sealed class TrackingConfiguration
         public string Key { get; set; }
 
         /// <summary>
+        /// The identity of the target. This severs to identify which stored data belongs to which object. If not specified,
```

Comment:
I can't help but nitpick about the naming. From the outside `configuration.IncludeClassName` seems kinda cryptic, it's not obvious what the property does. How about making this an enum instead of bool? Maybe something like: `StoreNamingScheme{ KeyOnly, KeyAndTypeName }` and the calling code would be `configuration.StoreNamingScheme = StoreNamingScheme.KeyOnly` or `IdentifyAs("blah", StoreNamingScheme.KeyOnly)`.